### PR TITLE
Create buildspec.yml

### PR DESCRIPTION
**Pull request by ki-ki-316**

### CODE REVIEW

File: buildspec.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  install:
+    runtime-versions:
+      nodejs: 16
+  build:
+    commands:
+      - echo This is ${ENV}
+      - echo ${URL} # defined in BuildProject"
+      - NEXT_PUBLIC_DESIGN_URL=${NEXT_PUBLIC_DESIGN_URL}
+      - NEXT_PUBLIC_HEALTH_API=${NEXT_PUBLIC_HEALTH_API}
+      - NEXT_PUBLIC_API_BASE_URI=${NEXT_PUBLIC_API_BASE_URI}
```
Comment (tatoMa) [inline buildspec.yml:13]: They may not need these environment variables as they use a different backend now.

File: buildspec.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+     # - NEXT_PUBLIC_GOOGLE_CLIENT_ID=${NEXT_PUBLIC_GOOGLE_CLIENT_ID}
+      - npm i next
```
Comment (tatoMa) [inline buildspec.yml:15]: Try only "npm i" to see if it working.

**Review by tatoMa — APPROVED** [review]
LGTM

File: buildspec.yml (hunk 2 above)
Comment (ki-ki-316) [reply buildspec.yml]: Yes, "npm i" its working

Comment (tatoMa) [reply buildspec.yml]: Nice work!

File: buildspec.yml (hunk 1 above)
Comment (ki-ki-316) [reply buildspec.yml:13]: Can you please let me know if there is anyone that I can confirm these with?

Comment (tatoMa) [reply buildspec.yml:13]: As far as I know, there would be no one needed in this repo at this point. But maybe it is better just to leave them there to see if they are needed in the future.

Comment (tatoMa) [reply buildspec.yml:13]: No change is needed at this point I believe.

Comment (ki-ki-316) [reply buildspec.yml:13]: OK, then I will leave them there. Thanks!

**Review by VanderDeng — APPROVED** [review]

**Pull request merged**